benchmarks/cpp/prepare_dataset.py

```diff
@@ -51,48 +52,65 @@ def get_tokenizer(cls,
     required=True,
     type=str,
     help=
-    "Tokenizer dir for the model run by gptManagerBenchmark, or the model name from HuggingFace."
+    "Tokenizer dir for the model run by gptManagerBenchmark, or the model name from HuggingFace.",
+)
+@click.option(
+    "--output",
+    type=str,
+    help="Output json filename.",
+    default="preprocessed_dataset.json",
 )
-@click.option("--output",
-              type=str,
-              help="Output json filename.",
-              default="preprocessed_dataset.json")
 @click.option(
     "--stdout",
     is_flag=True,
     help="Print output to stdout with a JSON dataset entry on each line.",
-    default=False)
-@click.option("--random-seed",
-              required=False,
-              type=int,
-              help="random seed for token_ids",
-              default=420)
+    default=False,
+)
+@click.option(
+    "--random-seed",
+    required=False,
+    type=int,
+    help="random seed for token_ids",
+    default=420,
+)
 @click.option("--task-id", type=int, default=-1, help="LoRA task id")
 @click.option("--rand-task-id",
               type=int,
               default=None,
               nargs=2,
               help="Random LoRA Tasks")
-@click.option("--log-level",
-              default="info",
-              type=click.Choice(['info', 'debug']),
-              help="Logging level.")
+@click.option(
+    "--log-level",
+    default="info",
+    type=click.Choice(["info", "debug"]),
+    help="Logging level.",
+)
+@click.option(
```

Comment:
We should update the performance guide since this changes the argument needed to create datasets for `trtllm-bench`.

Reply:
The changes as they are should be backwards compatible. Meaning that there is no **need** for `trtllm-bench` users to change their usage at this time. However, they are now given the **possibility** of using this API instead of going through stdout. At some point, we can mark `--stdout` as deprecated, and then remove it, but it can be done at a later point.

Reply:
Got it -- that invalidates one of my other comments. If we aren't already, maybe print a deprecation warning?

Reply:
Marked parameter as deprecated and added some more doc on it.